src/state-summary/state-card-weather.html

```diff
@@ -0,0 +1,110 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<link rel="import" href="../../bower_components/google-apis/google-legacy-loader.html">
+
+<dom-module id="state-card-weather">
+  <template>
```

Comment:
Yes indeed, I was planning on adding some more current information as well. Something like the shown example would be really nice (although I think OWM doesn't include all that information, like wind/precipitation forecast)